app/controllers/results_controller.rb

```diff
@@ -59,8 +59,10 @@ def edit
     @first_file = @files.first
     @extra_marks_points = @result.extra_marks.points
     @extra_marks_percentage = @result.extra_marks.percentage
-    @marks_map = Hash.new
-    @old_marks_map = Hash.new
+    @flexible_marks_map = Hash.new
```

Comment:
I made two sets because looking ahead we will need to show the marks in the summary pane for both flexible and rubric criteria. Flexible and rubric have different formats so we need to pass a list of flexible criteria to one partial and a list of rubric criteria to the other partial. Before we used to check for the marking_scheme type and either render the partial for flexible criteria or render the partial for rubric_criteria.